deploy-manage/security/secure-settings.md

```diff
@@ -16,232 +15,177 @@ mapped_pages:
+**Other examples**
 
-If the credential values are correct, but do not work, the keystore file could be out of sync on one or more nodes. To sync the keystore file, update the value for the key by using the patch API to delete the key from keystore, then add it back again.
-
-:::
+For a full command reference and additional examples, such as displaying stored values or adding entire files as setting values, refer to the [`elasticsearch-keystore` tool documentation](elasticsearch://reference/elasticsearch/command-line-tools/elasticsearch-keystore.md).
 
 ::::
 
-::::{tab-item} Self-managed
-:sync: self-managed
-
-All the modifications to the keystore take effect only after restarting {{es}}.
-
-These settings, just like the regular ones in the `elasticsearch.yml` config file, need to be specified on each node in the cluster. Currently, all secure settings are node-specific settings that must have the same value on every node.
-
+:::::
 
-**Reloadable secure settings**
+### Reloadable settings [reloadable-secure-settings]
 
-Just like the settings values in `elasticsearch.yml`, changes to the keystore contents are not automatically applied to the running {{es}} node. Re-reading settings requires a node restart. However, certain secure settings are marked as **reloadable**. Such settings can be re-read and applied on a running node.
+Just like the settings values in `elasticsearch.yml`, changes to the keystore contents are not automatically applied to the running {{es}} node. Re-reading settings requires a node restart.
 
-You can define these settings before the node is started, or call the [Nodes reload secure settings API](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-nodes-reload-secure-settings) after the settings are defined to apply them to a running node.
+However, certain secure settings are marked as **reloadable**. Such settings can be re-read and applied on a running node by using the [Nodes reload secure settings API](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-nodes-reload-secure-settings).
 
-The values of all secure settings, **reloadable** or not, must be identical across all cluster nodes. After making the desired secure settings changes, using the `bin/elasticsearch-keystore add` command, call:
+This API should be called after having properly defined the keystore settings in all nodes of the cluster. The values of all secure settings, **reloadable** or not, must be identical across all cluster nodes. After making the desired secure settings changes, call:
 
 ```console
 POST _nodes/reload_secure_settings
 {
   "secure_settings_password": "keystore-password" <1>
 }
 ```
-
-1. The password that the {{es}} keystore is encrypted with.
-
+1. If the {{es}} keystore is password-protected, include the `secure_settings_password` parameter in the request body with the keystore password.
```

Comment:
I'd probably put this in the previous instruction instead of as an annotation (this is a nitpick)